shard-operator/src/main/kubernetes/openshift.yml

```diff
@@ -29,6 +23,7 @@ rules:
     resources:
       - deployments
       - services
+      - namespaces
```

Comment:
Why is it needed to have rights for namespace?

Reply:
@sutaakar when a new resource of a new customer is requested the shard has to create a new namespace. It will be used for all the customer resources

Reply:
ok